src/encoded/static/components/viz/Workflow/Node.js

```diff
@@ -230,10 +234,6 @@ export default class Node extends React.Component {
         */
     }
 
-    static propTypes = {
-        'title' : PropTypes.func.isRequired
```

Comment:
We no longer need this, because output of `renderNodeElement(node, graphNodeProps)` (aka our WorkflowNodeElement) should handle all of that (titles on node).